docs/topics/jvm/jvm-records.md

```diff
@@ -61,14 +56,4 @@ To declare a data class with the `@JvmRecord` annotation, it must meet the follo
 * The class cannot declare any properties with backing fields – except those initialized from the corresponding primary constructor parameters.
 * The class cannot declare any mutable properties with backing fields.
 * The class cannot be local.
-* The primary constructor of the class must be as visible as the class itself.
-
-## Enabling JVM records
-
-To start using JVM records, add the compiler options `-Xjvm-enable-preview` and `-language-version 1.5`.
-
-See [compiler options in Gradle](gradle.md#compiler-options) for details.
-
-## Further discussion
-
-See this [language proposal for JVM records](https://github.com/Kotlin/KEEP/blob/master/proposals/jvm-records.md) for further technical details and discussion.
```

Comment:
The link to KEEP is still useful, because it contains some ideas for further improvements of record support and the place for their discussion.

Reply:
Rollbacked.